packages/client/src/views/SysAdmin/Config/Application/utils.ts

```diff
@@ -0,0 +1,221 @@
+/*
```

Comment:
Which of these are used in this PR? Maybe we could only keep those ones

Reply:
The `getCurrencySymbol` method that was used is relocated [here](https://github.com/opencrvs/opencrvs-core/blob/64e6732eee371025e006b0b2d66bc5ec859a440f/packages/client/src/utils/currencyUtils.ts#L29). So we don't need to keep this file